### DISCUSSION TITLE
Physiological fluid volume in the gut

### DISCUSSION THREAD

**Discussion by Boulenc** [Open-Systems-Pharmacology/Forum#893] [Questions & Problems] [answered]
I am struggling with bioparmaceutical aspects after oral route. Do you know if the the fluid volume in the gut can be changed?; what is the default value? I can change easily the volume of administration (3.5 mL/kg) in the ''Administration'' building block but not this physiological volume for human. This value should impact the solubilized dose fraction in the gut.
Thank you for your help.
Xavier

**Comment by StephanSchaller** [comment]
Hi Xavier,

you can change this once you have set up the simulation in the administration protocol (oral administration):
![image](https://user-images.githubusercontent.com/25477643/117151661-937e1200-adb9-11eb-866a-3e6fb26bc70d.png)

**Comment by Boulenc** [comment]
Hi Stephan,
OK for the administration volume, but I am talking about physiological fluid volume in the gut.  is it 0 by default?

**Comment by StephanSchaller** [comment, marked as answer]
Sorry, my misunderstanding...

You can change this via the fraction of stomach volume filled with liquid ... the parameter is called "Fractional steady state fill level":
It can be found here:
![image](https://user-images.githubusercontent.com/25477643/117154525-3768bd00-adbc-11eb-9cce-e62cb6b1e8d1.png)

The liquid volume is the product of "Fractional steady state fill level" and "Volume"

**Comment by Boulenc** [comment]
I Stephan,

Thank you.
Is 1.57 L the fluid volume ? of the total volume (empty and full) ?; it is high, BCS/DCS classification assume 250 mL and now 500 mL in total, with about 250 mL as water administration volume.
Can grey area be changed?

Regards

Xavier.


De : esqlabs GmbH ***@***.***>
Envoyé : mercredi 5 mai 2021 16:10
À : Open-Systems-Pharmacology/Forum ***@***.***>
Cc : Xavier Boulenc ***@***.***>; Author ***@***.***>
Objet : Re: [Open-Systems-Pharmacology/Forum] Physiological fluid volume in the gut (#728)


Sorry, my misunderstanding...

You can change this via the fraction of stomach volume filled with liquid ... the parameter is called "Fractional steady state fill level":
It can be found here:
[image]<https://ddec1-0-en-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fuser%2dimages.githubusercontent.com%2f25477643%2f117154525%2d3768bd00%2dadbc%2d11eb%2d9cce%2de62cb6b1e8d1.png&umid=47bf53b8-77f0-4f5d-8c64-6733832f9589&auth=670e6529b563b7dbb42ee90dda0d50ae13dc637b-613bfa7842c76f9d7b2b5615d9b34d5b5a74687b>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://ddec1-0-en-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fOpen%2dSystems%2dPharmacology%2fForum%2fissues%2f728%23issuecomment%2d832718675&umid=47bf53b8-77f0-4f5d-8c64-6733832f9589&auth=670e6529b563b7dbb42ee90dda0d50ae13dc637b-46a65f0f40f44f4a72ba0d05f73fdb0dcf9e28f2>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQ72QA7NN3RVSKAHZXXBXXTTMFGRVANCNFSM44EYMHBA>.

Veuillez trouver notre information relative à la protection des données ici<https://www.evotec.com/en/about/site-information/data-protection-fr>

STATEMENT OF CONFIDENTIALITY.

This email and any attachments may contain confidential, proprietary, privileged and/or private information.  
If received in error, please notify us immediately by reply email and then delete this email and any attachments from your system. Thank you.

**Comment by StephanSchaller** [comment]
Hi Xavier,

no, this is the luminal volume of a "full" stomach. 
Liquid volume is "Fractional steady state fill level" multiplied with this "Volume" .. so around 50 ml?
This would be the liquid volume in a fasted state. What the is added with the oral administration comes on top.

Best,
Stephan

**Comment by Boulenc** [comment]
Hi Stephan,

My turn of misunderstanding ☺
Clear ; thank you for your help.

Regards.

Xavier.


De : esqlabs GmbH ***@***.***>
Envoyé : mercredi 5 mai 2021 17:14
À : Open-Systems-Pharmacology/Forum ***@***.***>
Cc : Xavier Boulenc ***@***.***>; Author ***@***.***>
Objet : Re: [Open-Systems-Pharmacology/Forum] Physiological fluid volume in the gut (#728)


Hi Xavier,

no, this is the luminal volume of a "full" stomach.
Liquid volume is "Fractional steady state fill level" multiplied with this "Volume" .. so around 50 ml?
This would be the liquid volume in a fasted state. What the is added with the oral administration comes on top.

Best,
Stephan

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://ddec1-0-en-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fOpen%2dSystems%2dPharmacology%2fForum%2fissues%2f728%23issuecomment%2d832769998&umid=aa132d9d-5814-47af-9f84-f461848fa84a&auth=670e6529b563b7dbb42ee90dda0d50ae13dc637b-decdc912918a38edee1366af2def829f50086814>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQ72QAYOHZBUYUGHA6ZYXLTTMFOCNANCNFSM44EYMHBA>.

Veuillez trouver notre information relative à la protection des données ici<https://www.evotec.com/en/about/site-information/data-protection-fr>

STATEMENT OF CONFIDENTIALITY.

This email and any attachments may contain confidential, proprietary, privileged and/or private information.  
If received in error, please notify us immediately by reply email and then delete this email and any attachments from your system. Thank you.
